src/routes.tsx

```diff
@@ -63,6 +66,7 @@ export const routes: RouteObject[] = [
       {
         path: `/${paraTime}/transactions`,
         element: <TransactionsPage />,
+        errorElement: <RoutingErrorPage />,
```

Comment:
> The drawback is that there is no TypeScript-level test to verify that the errorElement (or some other error boundary) is always there

Is there a way to add a top-level error handler?

Reply:
Yes, in fact it's recommended by React Router. What should be the text? Just a generic unknown error?